bin/metrics-pdns.rb

```diff
@@ -28,36 +29,23 @@
 require 'sensu-plugin'
 require 'sensu-plugin/metric/cli'
 require 'English'
+require 'socket'
 
 class PdnsGraphite < Sensu::Plugin::Metric::CLI::Graphite
-  option :dump_file,
-         description: 'name of file to dump powerDNS stats to',
-         short: '-d DUMPFILE',
-         long: '--dump DUMPFILE',
-         default: 'tmp.txt'
-
-  def output(*args)
-    return unless args.empty?
-    args[2] = Time.now.to_is if args[2].nil?
-    puts "#{`hostname -f`.chomp}-- #{args[0..2].join('\s')}"
-  end
+  option :scheme,
+         description: 'Metric naming scheme, text to prepend to metric',
+         long: '--scheme SCHEME',
+         default: "#{Socket.gethostname}.pdns"
 
   def run
-    # dump powerdns statistics using rec control
-    # TODO: @haashah only extract user specified metrics.
```

Comment:
I would prefer to keep this TODO as its a useful feature to add to the scripts functionality

Reply:
Agree that it's useful functionality. Can you open an issue for this. That'll increase visibility and we can track the work.